docpages/example_programs/music_and_audio/oggopus.md

```diff
@@ -191,7 +191,7 @@ You can compile this example using the following command
 ## Using liboggz
 
 You can use `liboggz` to stream an Ogg Opus file to discord voice channel.
-`liboggz` provides higher level abstraction and useful APIs. Some API `liboggz` provide includes seeking and timestamp interpretation.
+`liboggz` provides higher level abstraction and useful APIs. Some APIs `liboggz` provide includes seeking and timestamp interpretation.
```

Comment:
This is still grammatically incorrect either way. Should be something like:
Some APIs (for example `liboggz`) .....

Reply:
No, it's specifically talking about liboggz, so it would be some features it provides?

Reply:
Ah right, I understand now. I wouldn't name that as "APIs" then, but rather "some features `liboggz` provides includes".

Reply:
Yeah, i was trying to make as small as changes as possible while making it correct